app/src/main/java/com/x8bit/bitwarden/data/auth/repository/util/SyncResponseJsonExtensions.kt

```diff
@@ -31,21 +34,30 @@ fun List<SyncResponseJson.Profile.Organization>.toOrganizations(): List<Organiza
  */
 val SyncResponseJson.Policy.policyInformation: PolicyInformation?
     get() = data?.toString()?.let {
+        var json = Json {
+            ignoreUnknownKeys = true
+            explicitNulls = false
+            serializersModule = SerializersModule {
+                contextual(ZonedDateTimeSerializer())
+            }
+            coerceInputValues = true
+        }
```

Comment:
@david-livefront thoughts on converting this to a member function of `PolicyManager`? Something like...
```kotlin
fun getPolicyInformationOrNull(policy: Policy): PolicyInformation?
```

Unless I'm overlooking one, all of the references to this ext. property are located in classes that already have `PolicyManager` injected. The exception to that is references in `PolicyManagerExtensions`, but it will inherently have access to the new member function.

Reply:
I think we can just leave this here, it does feel like this warrants it's own `Json` instance.

I don't think we need to make it that complicated though, we just need this:

```kotlin
val json = Json { ignoreUnknownKeys = true }
```

Reply:
P.S.
The purpose of making it a member function would be to prevent instantiating a new `json` object here. Forgot to mention that, for context. 😅

Reply:
We could make this a top level property to avoid making a bunch of them too.

Reply:
`explicitNulls` is also necessary on the json. 
`defaultValue` property name has changed and it missing now on the `data`